judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/ContestRoleChecker.java

```diff
@@ -48,8 +48,10 @@ public ContestRole getRole(String userJid, Contest contest) {
             return ContestRole.MANAGER;
         } else if (canSupervise(userJid, contest)) {
             return ContestRole.SUPERVISOR;
+        } else if (contestRoleDao.isContestant(userJid, contest.getJid())) {
```

Comment:
Please also add (the missing) integration test for this method, in `ContestRoleCheckerIntegrationTests`.